src/librustc/session/config.rs

```diff
@@ -410,6 +412,7 @@ top_level_options!(
 
         // Remap source path prefixes in all output (messages, object files, debug, etc)
         remap_path_prefix: Vec<(PathBuf, PathBuf)> [UNTRACKED],
+        edition: Edition [UNTRACKED],
```

Comment:
I think this needs to be TRACKED?

this is for incremental compilation; changing this should probably force a recompile

Reply:
Ah, so that's the the TRACKED/UNTRACKED thing means. Cool. I'll change this.